devices/MiHumidifierCB1.js

```diff
@@ -13,4 +15,17 @@ module.exports = class extends MiHumidifierCA1 {
             return temperature;
         };
     }
+
+    // TODO: find out if it is supported on other devices as well; only tested on CB1 yet
+    initializeMute() {
+        this.buzzerGetName = "buzzer";
+        this.convertBuzzerToMute = function (buzzer) {
+            return buzzer !== 'on';
+        };
+        this.buzzerSetName = "set_buzzer";
+        this.convertMuteToBuzzer = function (mute) {
+            return mute ? 'off' : 'on';
+        };
+    }
+
```

Comment:
buzzer is supported on all devices staring from v1, so I think this can be moved to V1 file (I have v1 model and checked it)

Reply:
Moved to MiHumidifierV1